sufia.gemspec

```diff
@@ -49,6 +49,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'flipflop', '~> 2.2'
   spec.add_dependency 'jquery-datatables-rails', '~> 3.4.0'
   spec.add_dependency 'rdf-rdfxml'
+  spec.add_dependency 'sprockets-rails', '~> 3.1.1'
```

Comment:
3.2 is out and travis was going bananas on it.

@jcoyne I think your hesitation is well-founded; it doesn't seem like anything has changed recently but this is also a slow-moving branch and it's possible no one has pushed to it since 3.2 wasa released. But I am not able to pursue a better fix for this right now. We can either leave the build broken or maybe ticket this issue and accept this PR so that this branch can move forward. I could split the PR into pieces according to whatever agrees with people, and override the travis check to leave the branch at least less broken. Open to whatever @projecthydra/sufia-code-reviewers prefer.